contrib/checkbox-ce-oem/checkbox-provider-ce-oem/units/mtd/jobs.pxu

```diff
@@ -9,7 +9,13 @@ plugin: shell
 user: root
 flags: also-after-suspend
 estimated_duration: 5
```

Comment:
```suggestion
estimated_duration: 5
requires: manifest.has_mtd == 'True'
imports: from com.canonical.plainbox import manifest
```

Reply:
I suggest that with use manifest in this job as well.

Reply:
Great idea!